_playbooks/playbook-pam.md

```diff
@@ -0,0 +1,367 @@
+    - text: Appendix A. Reference Documents
+      href: '#appendix-a-reference-documentation'
+    - text: Appendix B. Privileged User Agreement
+      href: '#appendix-b-privileged-user-agreement'
+    - text: Appendix C. 800-53 Privileged User Overlay
+      href: '#appendix-c-nist-sp-800-53-privileged-user-overlay'
+      
+---
+
+{% assign categories = "" | split: "" %}
+{% for pam in site.data.overlay_pam %}
+  {% assign category = pam.category | strip %}
+  {% assign categories = categories | push: category | uniq | sort %}
+{% endfor %}
+{% assign categories = categories | uniq | sort %}
+
+<a href="../../assets/img/logo-gsa.png" target="_blank" rel="noopener noreferrer"><img src="../../assets/img/logo-gsa.png" width="64" height='64' align="left" alt="U.S. General Services Administration Logo"></a>
+<a href="../../assets/img/logo-cio.png" target="_blank" rel="noopener noreferrer"><img src="../../assets/img/logo-cio.png" width="64" height='64' align="left" alt="U.S. Federal Chief Information Officer Council Logo"></a>
+<a href="../../assets/img/logo-cisa.png" target="_blank" rel="noopener noreferrer"><img src="../../assets/img/logo-cisa.png" width="64" height='64' align="left" alt="U.S. Department of Homeland Security Cybersecurity and Infrastructure Security Agency Logo"></a><br><br><br>
+
+This playbook is a collaboration between the General Services Administration (GSA) through the Federal CIO Council and the Department of Homeland Security (DHS) Cybersecurity and Infrastructure Security Agency (CISA)
+  
+Version 1.0  
+Last Updated: August xx, 2021
+
+# Executive Summary
+The Privileged User Playbook is a practical guide to **help federal agencies implement and manage a privileged user program**. Privileged users are a special user type that perform security-related duties. A privileged user program enables an agency to identify, track, monitor, and audit privileged users and accounts, effectively decreasing the cyber risk and impact on an agency's mission. **Unwanted behavior or compromise of privileged accounts are also responsible for the most high-profile federal security breaches**. It is a critical Identity, Credential, and Access Management capability to identify privileged users and access to high value assets. As part of agency insider threat programs, agencies can use this playbook to define privileged user program requirements and capabilities.
+
+There are three main use cases to identify a privileged account or user.
+1. Accounts used to manage IT infrastructure, resources of high value assets (HVA), and core systems (e.g. perform maintenance activities on human resource applications or databases).
+2. Help desk personnel with escalated privilege to perform security-relevant processes on endpoints (e.g. install software on user laptops or change endpoint configuration settings).
+3. Managers that approve access or accounts (e.g. a manager approving access requests to a risk reporting application).
```

Comment:
Change this to just a manager approving or recertifying access to an application.

Reply:
The access / account approval process includes approval by the Information Systems Security Manager (ISSM) as well as Physical Security Manager for clearance level verification in the DoD.